lab-jason/test/greet-test.js

```diff
@@ -0,0 +1,13 @@
+const assert = require('assert');
+const greet = require('../lib/greet');
+
+describe('greet', function(){
+  describe('#greet', function(){
+    it(`should return "hello ,<name>" with name being passed variable.`, function (){
+      assert.strictEqual(greet.greet('world!'), 'hello world!');
+    });
+    it(`should return null without variable.`, function(){
```

Comment:
Nitpick: I'd say "without argument" rather than "without variable".

* Arguments are values passed into functions.
* Parameters are what argument values are called inside a function.
* Variables are named values that aren't arguments (either local or global; declared with `var`/`let`).